code/modules/ninja/suit/ninjaDrainAct.dm

```diff
@@ -0,0 +1,270 @@
+
+/*
+
+Contents:
+- Assorted ninjaDrainAct() procs
+- What is Object Oriented Programming
+
+They *could* go in their appropriate files, but this is supposed to be modular
+
+*/
+
+
+//Needs to return the amount drained from the atom, if no drain, 0
+/atom/proc/ninjaDrainAct()
```

Comment:
Consistency error:

All other act procs are "x_act()"
this should be "ninjadrain_act()"